provider/python/README.md

```diff
@@ -0,0 +1,120 @@
+# Python capability provider for wasmCloud
+
+This capability provider allows wasmCloud actors to invoke functions in
+Python. It can be used to run Tensorflow, Keras, PyTorch, and other machine
+learning models in Python. Because it runs on a native host, it can take
+advantage of C libraries, including the [Intel Optimization for Tensorflow](https://www.intel.com/content/www/us/en/developer/articles/guide/optimization-for-tensorflow-installation-guide.html),
+GPUs, and any other Python libraries and extensions.
+
+This differs from other wasmCloud capability providers in a few
+significant ways:
+
+- There is not a smithy-defined interface contract. To invoke this
+  provider, an actor passes an operation name and a serialized
+  data payload. These are converted to Python objects and passed to a
+  dispatch handler function, described below.
+- Because of the limitations of the Python Global Interpreter Lock
+  (GIL), the provider is (mostly) single threaded. Some attempt is made 
+  to increase throughput by using threads and async IO for message handling and
+  serialization, but the Python functions are invoked one-at-a-time.
+- Running this provider requires a Python 3 environment to be already
+  configured on the host machine. Although the wasmCloud host could
+  download and run the provider from a remote registry, the host
+  must be configured as described below.
+
```

Comment:
Please add a caution sign and caveat here that this is an example for illustrative purposes, and that there are a number of extremely large security and stability issues here and this is not meant to be run in production.

Reply:
done